pir/cpp/benchmark.cpp

```diff
@@ -14,10 +14,11 @@ namespace pir {
 
 using namespace ::testing;
 
+constexpr bool USE_CIPHERTEXT_MULTIPLICATION = true;
 constexpr uint32_t ITEM_SIZE = 288;
 constexpr uint32_t DIMENSIONS = 2;
-constexpr uint32_t POLY_MOD_DEGREE = 8192;
-constexpr uint32_t PLAIN_MOD_BITS = 45;
+constexpr uint32_t POLY_MOD_DEGREE = 4096;
+constexpr uint32_t PLAIN_MOD_BITS = 24;
```

Comment:
What's the reason for changing the defaults? maybe we want to document that

Reply:
It would also be great if we had some references/formulas to how we obtained these constants.

Reply:
Created #43